ui/src/app/components/help-sidebar.tsx

```diff
@@ -582,6 +591,62 @@ export const HelpSidebar = fp.flow(
       </FlexRow>;
     }
 
+    withinPastTwentyFourHours(epoch) {
+      const completionTimeMoment = moment(epoch);
+      const twentyFourHoursAgo = moment().subtract(1, 'days');
+      return completionTimeMoment.isAfter(twentyFourHoursAgo);
+    }
+
+    displayExtractionIcon(icon: IconConfig) {
+      const {extractionJobs} = this.state;
+      const jobsByStatus = fp.groupBy('status', extractionJobs);
+      let status;
+      // If any jobs are currently running, show running icon.
+      if (jobsByStatus[TerraJobStatus.RUNNING]) {
+        status = TerraJobStatus.RUNNING;
+      } else if (
+          jobsByStatus[TerraJobStatus.FAILED]
+          && fp.some(job => this.withinPastTwentyFourHours(job.completionTime),
+            jobsByStatus[TerraJobStatus.FAILED]
+          )
+      ) {
+        status = TerraJobStatus.FAILED;
+      } else if (
+          jobsByStatus[TerraJobStatus.SUCCEEDED]
+          && fp.some(job => this.withinPastTwentyFourHours(job.completionTime),
+            jobsByStatus[TerraJobStatus.SUCCEEDED]
+          )
+      ) {
+        status = TerraJobStatus.SUCCEEDED;
+      }
+
+      // We always want to show the DNA icon.
+      // When there are running jobs, we will show a small overlay icon in the bottom right of the
+      // tab showing the job status.
+      return <FlexRow style={{height: '100%', alignItems: 'center', justifyContent: 'space-around'}}>
+        {this.displayFontAwesomeIcon(icon)}
+        <FlexRow data-test-id='extraction-status-icon-container' style={styles.statusIconContainer}>
+          {
+            switchCase(status,
+              [TerraJobStatus.RUNNING, () => <FontAwesomeIcon icon={faSyncAlt} style={{
+                ...styles.runtimeStatusIcon,
+                ...styles.rotate,
+                color: colors.runtimeStatus.starting,
```

Comment:
nit: should we generalize the naming of `colors.runtimeStatus` now that there's a second consumer? Some ideas: `colors.operation`, `colors.asyncOperation`, `colors.asyncStatus`

Reply:
the semantics are different for this - but I can duplicate some with the correct semantics.